include/net/nrf_cloud_location.h

```diff
@@ -57,6 +57,19 @@ struct nrf_cloud_location_result {
 	enum nrf_cloud_error err;
 };
 
+/** @brief Location request config */
+struct nrf_cloud_location_config {
+	/** If true, nRF Cloud will send the location back to the device. */
+	bool do_reply;
+	/** If true, uncertainty of result will be 95%, otherwise 68%. */
+	bool hi_conf;
+	/** If true, and location cannot be found, result will fall back to rough estimate
+	 *  based on the cell tower tracking area code.  If false, will return an error
+	 *  if the cloud cannot provide a higher accuracy response.
+	 */
+	bool fallback;
```

Comment:
Is this somehow related to `hi_conf` as it refers to `higher accuracy response`?

Reply:
No, according to Cole fallback is different. "Higher accuracy response" means the location based on the serving cell plus 0 or more neighbors or GCI cells. "Rough estimate" is less accurate than even a single cell tower location.

He said: "TAC is less accurate than single cell. TAC is unique within a single carrier can cover several towers. TAC is thousands of meters while single cell is hundreds of meters."

Reply:
I guess we don't know from the returned result whether fallback was used, right? But it would be visible as a higher accuracy value?

Reply:
Correct, the returned result does not specify that a fallback to a more coarse position was done, but if fallback occurs, the position will be *lower* accuracy (location based only on TAC instead of one or more specific cell towers).